Resources/Prototypes/_Goobstation/Changeling/Entities/Mobs/headcrab.yml

```
@@ -0,0 +1,46 @@
+      Dead:
+        Base: headcrab_dead
+  - type: MobThresholds
+    thresholds:
+      0: Alive
+      50: Dead
+  - type: MovementSpeedModifier
+    baseWalkSpeed: 3
+    baseSprintSpeed: 7
+  - type: ExplosionResistance
+    damageCoefficient: 0
+# - type: VentCrawler # ! UNCOMMENT WHEN PORTED
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)